packages/@aws-cdk/aws-cloudwatch/README.md

```diff
@@ -97,6 +142,25 @@ The most important properties to set while creating an Alarms are:
 - `evaluationPeriods`: how many consecutive periods the metric has to be
   breaching the the threshold for the alarm to trigger.
 
+### A note on units
+
+In CloudWatch, Metrics datums are emitted with units, such as `seconds` or
```

Comment:
I precisely picked "datums" to set it apart from "data" as "data" is used so much in our field that its meaning has become diffuse.

"Datum" has a very specific and well-defined meaning in this context, and in case you are wondering, it *is* an acceptable pluralization of "datum".

Reply:
I had checked... It is ugly to me, but I don't care all that much.